client/js/components/support/DpSupportCard.vue

```diff
@@ -77,14 +74,14 @@ export default {
     reachability: {
       required: false,
       type: Object,
-      default: () => {}
+      default: () => ({})
```

Comment:
why you did this here? its just for me for understanding. which error you get here or which problem?

Reply:
without the Brackets, You return undefined. With the brackets, You return an empty Object